config/arc_cluster/README.md

```diff
@@ -0,0 +1,37 @@
+# Running open-gira on ARC
+
+## Persistence
+
+Either create or attach to a tmux session:
+`tmux attach-session -t <id>`
+`tmux`
+
+## Resources
+
+Allocate some nodes for use:
+`salloc --ntasks-per-node=<max tasks per node> --nodes=<num nodes> --partition=<short|medium|long> --time=01:00:00 --mem=8000`
+
+## Python environment
+
+We need the snakemake executable to create jobs for us. We could get this from
+the module load system, but their version is quite old (6.10.0). N.B. Versions
+<7.0.0 may cause the following problem:
+https://github.com/snakemake/snakemake/issues/1392
+
+Instead use a conda environment we have created which contains snakemake:
+`conda activate snakemake-7.12.1`
+
+## Osmium
+
+N.B. Some open-gira jobs require the use of osmium. This has been compiled on
+the cluster with `$DATA/osmium/build_osmium.sh`. The executable (actually a
+wrapper) is in ~/bin. N.B. To _run_ osmium, the Boost C++ libraries must be
```

Comment:
Is osmium in your home directory at `~/bin`? Does each user need to build it, or could it go in a shared location?

Reply:
Resolved, see above